backend/app/api/admin_routes/upload.py

```diff
@@ -33,6 +34,14 @@ def upload_files(
                 status_code=status.HTTP_400_BAD_REQUEST,
                 detail="File name cannot be empty",
             )
+        if SiteSetting.setting_exists("upload_max_body_size"):
+            sys_upload_max_body_size = SiteSetting.get_setting("upload_max_body_size")
+            if file.size > sys_upload_max_body_size:
```

Comment:
As the `__getattr__` method has been overrided,

https://github.com/pingcap/autoflow/blob/32526871db6b4e92dacf37551b5f527964e8e41f/backend/app/site_settings/__init__.py#L72

You can just:

```suggestion
        sys_upload_max_body_size = SiteSetting.upload_max_body_size
        if sys_upload_max_body_size:
            if file.size > sys_upload_max_body_size:
```